CommunityToolkit.Authentication.Uwp/WebAccountProviderConfig.cs

```diff
@@ -19,15 +21,33 @@ public struct WebAccountProviderConfig
         /// </summary>
         public WebAccountProviderType WebAccountProviderType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the properties that need to be added when constructing <see cref="Windows.Security.Authentication.Web.Core.WebTokenRequest"/> (for MSA).
+        /// </summary>
+        public IDictionary<string, string> MSATokenRequestProperties { get; set; }
+
+        /// <summary>
+        /// Gets or sets the properties that need to be added when constructing <see cref="Windows.Security.Authentication.Web.Core.WebTokenRequest"/> (for AAD).
+        /// </summary>
+        public IDictionary<string, string> AADTokenRequestProperties { get; set; }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="WebAccountProviderConfig"/> struct.
         /// </summary>
         /// <param name="webAccountProviderType">The types of accounts providers that should be available to the user.</param>
         /// <param name="clientId">The registered ClientId. Required for AAD login and admin consent.</param>
-        public WebAccountProviderConfig(WebAccountProviderType webAccountProviderType, string clientId = null)
+        /// <param name="msaTokenRequestProperties">Request properties for MSA.</param>
+        /// <param name="aadTokenRequestProperties">Request properties for AAD.</param>
+        public WebAccountProviderConfig(
+            WebAccountProviderType webAccountProviderType,
+            string clientId = null,
+            IDictionary<string, string> msaTokenRequestProperties = null,
+            IDictionary<string, string> aadTokenRequestProperties = null)
         {
             WebAccountProviderType = webAccountProviderType;
             ClientId = clientId;
+            MSATokenRequestProperties = msaTokenRequestProperties;
+            AADTokenRequestProperties = aadTokenRequestProperties;
```

Comment:
Do we need to bother having these as options in the constructor since they're public mutable properties? Folks could just do

```cs
var wapc = new WebAccountProviderConfig(WebAccountProviderType.Whatever) {
   MSATokenRequestProperties = new() { { "Key", "Value" } }
};
```

Reply:
Added, thanks for suggestion